src/PaymentManager/Payment/PayU.php

```diff
@@ -180,7 +180,7 @@ public function initPayment(PriceInterface $price, array $config)
     /**
      * @param array $items
      *
-     * @return OnlineShopOrderItem[]
+     * @return array
```

Comment:
If we are going to keep the doc block return type, this would make more sense than just an array
```suggestion
     * @return OnlineShopOrderItem[]
```

Reply:
Yes, I agree it would make for sense to keep the type, instead of a plain array